clang/include/clang/StaticAnalyzer/Core/PathDiagnosticConsumers.h

```diff
@@ -1,4 +1,5 @@
-//===--- PathDiagnosticConsumers.h - Path Diagnostic Clients ------*- C++ -*-===//
+//===--- PathDiagnosticConsumers.h - Path Diagnostic Clients ------*- C++
+//-*-===//
```

Comment:
```suggestion
//===--- PathDiagnosticConsumers.h - Path Diagnostic Clients ----*- C++ -*-===//
```
Uhh, I hope I didn't miss similar cases.